tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py27,py34,py35,flake8,cov
+
+[testenv]
+commands = py.test {posargs}
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    flake8: python2.7
+    cov: python2.7
```

Comment:
I guess that's fine. The `test-results.xml` will be overwritten multiple times though. If that doesn't bother jenkins/travis…